README.md

```diff
@@ -236,17 +236,19 @@ export default Component;
 
 **The following environment variables are embedded during the build time.**
 
-* `process.env.RAZZLE_PUBLIC_DIR`: Path to the public directory.
+* `process.env.RAZZLE_PUBLIC_DIR`: Absolute path to the public directory in the server's filesystem.
 * `process.env.RAZZLE_ASSETS_MANIFEST`: Path to a file containing compiled asset outputs
 * `process.env.REACT_BUNDLE_PATH`: Relative path to where React will be bundled during development. Unless you are modifying the output path of your webpack config, you can safely ignore this. This path is used by `react-error-overlay` and webpack to power up the fancy runtime error iframe. For example, if you are using common chunks and an extra entry to create a vendor bundle with stuff like react, react-dom, react-router, etc. called `vendor.js`, and you've changed webpack's output to `[name].js` in development, you'd want to set this environment variable to `/static/js/vendor.js`. If you do not make this change, nothing bad will happen, you will simply not get the cool error overlay when there are runtime errors. You'll just see them in the console. Note: This does not impact production bundling.
 * `process.env.VERBOSE`: default is false, setting this to true will not clear the console when you make edits in development (useful for debugging).
-* `process.env.PORT`: default is `3000`, unless changed
-* `process.env.HOST`: default is `0.0.0.0`
+* `process.env.PORT`: The `BUILD_TARGET=server` build listens on this port for all NODE_ENVs. default is `3000`
+* `process.env.PORT_DEV`:  The `NODE_ENV=development` `BUILD_TARGET=client` listens on this port. Unused when `NODE_ENV=production`. default is `3001`
```

Comment:
Let’s discuss the naming convention here.  Im not fan of PORT_DEV. Should probably be like HMR_PORT or something.

Reply:
Looking back, I didn't even use that var and assumed PORT+1. I'll remove it unless there's any good reason to have it.

Reply:
Someone else added PORT_DEV
https://github.com/jaredpalmer/razzle/commit/f979863ff391ddb6df9c694e6e0ebb054840981c#diff-7c88c5c3855db79976ff110d7e59c9fc
Looks like I was carrying that over from packages/razzle/README.md

Reply:
Hmm must have missed that. We should probably rename it.

Reply:
In a different PR though, right?